libyul/YulControlFlowGraphExporter.h

```diff
@@ -28,15 +28,16 @@ using namespace yul;
 class YulControlFlowGraphExporter
 {
 public:
-	YulControlFlowGraphExporter(ControlFlow const& _controlFlow);
+	YulControlFlowGraphExporter(ControlFlow const& _controlFlow, ControlFlowLiveness const& _liveness);
```

Comment:
The `export*` functions are implemented like liveness is optional, the constructor of the exporter is implemented so that it is unconditionally required. I'd just make it a `ControlFlowLiveness const*` here as well.